src/installer/pkg/projects/netcoreapp/pkg/legacy/Microsoft.NETCore.App.Internal.pkgproj

```diff
@@ -20,8 +20,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Dependency Include="Microsoft.NETCore.Platforms" Version="$(Version)" TargetFramework="$(NetCoreAppCurrent)" />
-    <Dependency Include="Microsoft.NETCore.Targets" Version="$(Version)" TargetFramework="$(NetCoreAppCurrent)" />
+    <Dependency Include="Microsoft.NETCore.Platforms" Version="$(MicrosoftNETCorePlatformsPackageVersion)" TargetFramework="$(NetCoreAppCurrent)" />
```

Comment:
What would happen when we're on a servicing wave when the .NETCore.* packages are not stable?

Reply:
We don't service these packages every-time  but true we have to manually update when we service these packages.
I can create an issue to track this.

Reply:
Should we use these properties as the property that drives the package version that we produce as well? I think we should provide a better solution than this... what do you think?